### PR TITLE
fix: generation issues

### PR DESCRIPTION
**Pull request by renancaraujo**
## Description
- Remove the copyright header from the version file in the brick
- Fix generator workflow
- Improve generated and root READMEs
- Changed default cli name from `my_cli` because it is already [published](https://pub.dev/packages/my_cli) and may result in some confusion with the version prompt.
- Changed the depth of the brick to not include a sub directory with the project name 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: tool/brick_generator/main.dart [hunk 1]
```diff
@@ -34,7 +34,10 @@ void main() async {
       var file = _;
 
       try {
-        if (p.extension(file.path) == '.dart') {
+        // Generated file, to dot create it with copyright header
```
Comment (felangel) [inline tool/brick_generator/main.dart:37]: Is this a typo? Not sure what to dot means haha

Comment (renancaraujo) [reply tool/brick_generator/main.dart]: @renancaraujo
[oh my dislexy](https://github.com/VeryGoodOpenSource/very_good_dart_cli/pull/3/commits/4107e3c6cd7308eb3a8311de1e0ff991915395a8)

File: README.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ A dart CLI template created by the [Very Good Ventures Team][very_good_ventures_
 
 Generated by the [Very Good CLI][very_good_cli_link] 🤖
 
+---
```
[the remark this answers is not in the document]
Comment (felangel) [reply README.md:16]: And can we update the h1 at the top of the README to just say:

`# Very Good Dart CLI`

File: tool/brick_generator/main.dart [hunk 3]
```diff
@@ -34,7 +34,7 @@ void main() async {
       var file = _;
 
       try {
-        // Generated file, to dot create it with copyright header
+        // Generated file, do not create it with copyright header
         final isVersionFile =
```
[the remark this answers is not in the document]
Comment (renancaraujo) [reply tool/brick_generator/main.dart:37]: Lol, I really don't like that negated name, so I changed the condition and kept the name to be really `isVersionFile`.

**Review by felangel — APPROVED** [review]

**Pull request merged**